src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -736,13 +736,7 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         /// This trainer can be used as a baseline for other more sophisticated mdels.
         /// </remarks>
         /// <param name="catalog">The <see cref="BinaryClassificationCatalog"/>.</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        ///  [!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/RandomTrainerSample.cs)]
-        /// ]]></format>
-        /// </example>
-        public static RandomTrainer Random(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog)
+        internal static RandomTrainer Random(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog)
```

Comment:
Can this just be deleted?

Reply:
Yeah... actually from what I'm reading of the prior documentation it is simply not necessary at all. As far as I can tell, it existed in pre-`IDataView` days when evaluation didn't happen by evaluating predictions (which are of course just data), but evaluation happened over predictors. Nowadays evaluation of course just takes an `IDataView`, so involving predictors is not necessary at all (and in fact merely complicates the whole story). Even from those days I can't even find any instances of that actually becoming useful, so while I was somewhat reluctant to undertake that drastic step, I think it probably is appropriate now that I've read a bit more about this.

Reply:
Keeping the predictor for now for potential back-compat (we can figure out what we really want to do about that later).